src/qt/pivx/forms/mninfodialog.ui

```diff
@@ -101,7 +101,7 @@
            <string notr="true">padding-left:24px;</string>
           </property>
           <property name="text">
-           <string>Master Node Information</string>
+           <string>Masternode information</string>
```

Comment:
This is actually a dummy string (it is set in the `.cpp` file to it's visible value). 

Rather than have a duplicate string here, would change this line to read
```html
<string notr="true">N/A</string>
```

Reply:
Since this is a static title, I think it would be preferable to do the other way around: set it here once and avoid the duplication in the .cpp file.